easybuild/easyconfigs/b/batchgenerators/batchgenerators-0.25-foss-2021a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'batchgenerators'
+version = '0.25'
+
+homepage = 'https://github.com/MIC-DKFZ/batchgenerators'
+description = """Data augmentation toolkit developed at the Division of Medical
+Image Computing at the German Cancer Research Center (DKFZ) to suit all our
+deep learning data augmentation needs."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['38a67413e847ff367e64abac36331fcb065494202d526ae96f7644de3a0e5495']
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('scikit-image', '0.18.3'),
+    ('scikit-learn', '0.24.2'),
+]
+
+# remove requirement on Pillow and unittest2, which are not actually used
+preinstallopts = "sed -i '13d;19d' setup.py &&"
+
```

Comment:
I think this can add
runtest = 'pytest'

Reply:
the tarball from PyPI does not have the tests :cry: